README.md

```diff
@@ -4,6 +4,7 @@
 _The following setup is known to be working well. You can try other versions at your 
 own risk of losing a lot of time._
 * [Visual Studio 2015 Build Tools](http://landinghub.visualstudio.com/visual-cpp-build-tools) *
+* [OpenFin runtime executables](https://developer.openfin.co/versions/?product=Runtime&version=stabe) 
```

Comment:
Should this be version= or version=stable?  All three yield the same results but in case the handling of version=stable changes at some point?

Reply:
@bingenito Thank you, fix in #336